src/main/webapp/search_area.js

```diff
@@ -103,11 +129,24 @@ class SearchArea {
         newOrganization.popupElement.remove();
       });
 
-      this.organizationList.appendChild(newOrganization.getOrganization());
+      this.organizationListDiv.appendChild(newOrganization.getOrganization());
     });
+  }
 
-    this.organizationSearchArea.appendChild(this.organizationList);
-    this.searchArea.appendChild(this.organizationSearchArea);
-    this.searchArea.appendChild(this.organizationPopupArea);
+  async handleZipcodeSubmission(zipcodeForm) {
+    this.filterParams.set("zipcode", zipcodeForm.zipcode.value);
+    this.organizationsObjectsList = [];
+    this.organizationListDiv.innerHTML = "";
```

Comment:
use .textContent it is more secure than .innerHTML

Reply:
I used innerHTML here bc the last time I implemented code that clears existing divs like this I was told to change it to innerHTML instead of textContent